### PR TITLE
Add Ververica Charts

### PR DESCRIPTION
**Pull request by austince**
Original creators of Apache Flink, more about them here: https://www.ververica.com/

### CODE REVIEW

File: repos.yaml [hunk 1]
```diff
@@ -976,3 +976,8 @@ repositories:
     maintainers:
       - name: Sean Krail
         email: sean.krail@jitterbit.com
+  - name: ververica
+    url: https://charts.ververica.com
+    maintainers:
+      - name: Ververica
+        email: support@ververica.com
```
Comment (mattfarina) [inline repos.yaml]: @austince it doesn't appear you work for Ververica. Is there someone who works there who can vouch that they want this listed?

Comment (austince) [reply repos.yaml]: No, we're a customer - I'll reach out to them and get approval.

File: repos.yaml [hunk 2]
```diff
@@ -976,3 +976,8 @@ repositories:
     maintainers:
       - name: Sean Krail
         email: sean.krail@jitterbit.com
+  - name: ververica
+    url: https://charts.ververica.com
```
Comment (patricklucas) [inline repos.yaml]:
```suggestion
    url: https://charts.ververica.com/
```

File: repos.yaml (hunk 1 above)
Comment (patricklucas) [inline repos.yaml]:
```suggestion
        email: platform@ververica.com
```

**Comment by patricklucas** [comment]
Just a note that Ververica Platform is proprietary software that requires a license to use, but I do not see any restrictions on Helm Hub to that point and we're otherwise happy to be listed.

**Comment by austince** [comment]
Thanks @patricklucas - rebasing and signing now!

**Pull request merged**